lib/plugins/leetcode.js

```diff
@@ -594,27 +620,50 @@ plugin.githubLogin = function(user, cb) {
       if (resp.statusCode !== 200) {
         return cb('GitHub login failed');
       }
-      _request.get({url: leetcodeUrl}, function(e, resp, body) {
-        const redirectUri = resp.request.uri.href;
-        if (redirectUri !== 'https://leetcode.com/') {
-          return cb('GitHub login failed or GitHub did not link to LeetCode');
+      if (resp.request.uri.href === urls.github_tf_redirect) {
+        cb('Your GitHub are using two-factor authentication');
```

Comment:
Emm, why we need to call `cb` here? Can we just log a message with info level?

Reply:
I was using it  for test in vscode-leetcode if it can use two-factor, I will delete it.